go/mobile/identity.go

```diff
@@ -0,0 +1,208 @@
+package iden3mobile
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	ethkeystore "github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/iden3/go-iden3-core/components/idenpuboffchain/readerhttp"
+	"github.com/iden3/go-iden3-core/components/idenpubonchain"
+	"github.com/iden3/go-iden3-core/db"
+	"github.com/iden3/go-iden3-core/eth"
+	"github.com/iden3/go-iden3-core/identity/holder"
+	babykeystore "github.com/iden3/go-iden3-core/keystore"
+	"github.com/iden3/go-iden3-crypto/babyjub"
+	log "github.com/sirupsen/logrus"
+)
+
+type Identity struct {
+	id          *holder.Holder
+	Tickets     *TicketsMap
+	eventSender Event
+	storage     db.Storage
+}
+
+// TODO:
+// store on changes vs store on stop
+
+// NewIdentity creates a new identity
+// this funciton is mapped as a constructor in Java.
+// NOTE: The storePath must be unique per Identity
+func NewIdentity(storePath, pass, web3Url string, checkTicketsPeriodMilis int, extraGenesisClaims *BytesArray, e Event) (*Identity, error) {
+	id := &Identity{}
```

Comment:
No need to do this.  To return error, just do `return nil, err`

Reply:
Fixing it